c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -6,6 +6,9 @@ Version 1.6
 
 1. Get the new and heavily modified files:
 
+    .build/venv/bin/pcreate --interactive -s c2cgeoportal_create /tmp/{{project}} package={{package}} srid=-1
+    .build/venv/bin/pcreate --interactive -s c2cgeoportal_update /tmp/{{project}} package={{package}}
```

Comment:
unclear ? 
Do you follow this doc ?
http://docs.camptocamp.net/c2cgeoportal/1.6/integrator/upgrade_application.html#easy-upgrading-an-application-from-1-5-to-1-6